ogusa/get_micro_data.py

```diff
@@ -129,7 +130,7 @@ def get_data(
     """
     # Compute MTRs and taxes or each year, but not beyond TC_LAST_YEAR
     lazy_values = []
-    for year in range(start_year, TC_LAST_YEAR + 1):
+    for year in range(start_year, start_year + BW):
```

Comment:
Having the user specify the budget window here, rather than just setting the BW to the maximum available years (i.e., `TC_LAST_YEAR - start_year`), is probably a good idea.  But you might add an `assert` statement to ensure that `start_year + BW < TC_LAST_YEAR`.

Also, should it be `range(start_year, start_year + BW + 1)` since `range` isn't inclusive of the last element?